.github/workflows/build.yml

```diff
@@ -1,49 +1,164 @@
-name: Reusable WF - Build
+name: Build
+run-name: "Branch Build (${{ github.run_attempt }}.${{ github.run_number }}) - ${{ github.ref_name }}"
 
 on:
-  workflow_call:
+  workflow_dispatch:
     inputs:
-      version:
-        required: true
+      amalgam-lang-py-build:
+        description: |
+          (Optional) amalgam-lang-py build or version number. Defaults to pinned release.
+          
+          Examples:
+            '6807310618',
+            'https://github.com/howsoai/amalgam-lang-py/actions/runs/6807310618',
+            '4.0.1'
+        required: false
         type: string
-      payload:
+      howso-engine-py-build:
+        description: |
+          (Optional) howso-engine-py build or version number. Defaults to pinned release.
         required: false
         type: string
+  workflow_call:
+    inputs:
       build-type:
         required: false
         type: string
+      optional-release-tag:
+        required: false
+        type: string
 
 defaults:
   run:
     shell: bash
 
 jobs:
 
+  metadata:
+    uses: howsoai/.github/.github/workflows/set-metadata.yml@main
+    secrets: inherit
+    with:
+      # Remove unused parameters
+      build-type: ${{ inputs.build-type }}
+      optional-release-tag: ${{ inputs.optional-release-tag }}
+      amalgam-lang-py-build: ${{ inputs.amalgam-lang-py-build }}
+      howso-engine-py-build: ${{ inputs.howso-engine-py-build }}
+
   workflow-summary:
-    if: inputs.build-type != 'release'
-    uses: "./.github/workflows/workflow-summary.yml"
+    needs: ['metadata']
+    uses: "howsoai/.github/.github/workflows/workflow-summary.yml@main"
+    secrets: inherit
+    with:
+      upstream-details: ${{ needs.metadata.outputs.upstream-details }}
+
+  pytest-linux-3-8-st:
+    if: inputs.build-type != 'PR'
```

Comment:
Same reason as the other recipes: processing time is very long?

Reply:
Indeed (though less so than synth recipes/BDD). Talked with @cademack about this a bit -- I think we're going to push to make a habit of releasing recipes frequently instead of just relying on the nightly.